test/quantization/test_quantize_fx.py

```diff
@@ -1347,6 +1347,20 @@ def test_conv_functional_bias_not_observed(self):
                 prepare_expected_node_occurrence=expected_node_occurrence,
             )
 
+    def test_linear_functional_bias_not_observed(self):
```

Comment:
can we enable the functional test in test_linear? it was not enabled because of some issues in fx which was fixed now I think

Reply:
sure, did this in https://github.com/pytorch/pytorch/pull/49740 to keep PRs small